lib/qonfig/commands/load_from_self.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+module Qonfig
+  module Commands
+    # @api private
+    # @since 0.2.0
+    class LoadFromSelf < Base
+      # @return [String]
+      #
+      # @api private
+      # @since 0.2.0
+      attr_reader :caller_location
+
+      # @param caller_location [String]
+      #
+      # @api private
+      # @sicne 0.2.0
+      def initialize(caller_location)
+        @caller_location = caller_location
+      end
+
+      # @param settings [Qonfig::Settings]
+      # @return [void]
+      #
+      # @api private
+      # @since 0.2.0
+      def call(settings)
+        caller_file = caller_location.split(':').first
+        data_match = IO.read(caller_file).match(/\n__END__\n(?<end_data>.*)/m)
```

Comment:
You may allow whitespaces after `__END__` (with `/\n__END__\s*?\n(?<end_data>.*)/m`)

Reply:
Or just anything with `.*?`

Reply:
I think this is useless cuz Ruby language requires `\n__END__\n` annotation for the `__END__`-functionality. Otherwise you will get corresponding errors:

```ruby

"__END__   anything"
NameError: undefined local variable or method `anything' for main:Object

"__END__   "
NameError: undefined local variable or method `__END__' for main:Object
```

Reply:
Okay :(